crates/bevy_render/src/camera/camera.rs

```diff
@@ -76,6 +76,36 @@ pub struct ComputedCameraValues {
     old_viewport_size: Option<UVec2>,
 }
 
+#[derive(Component)]
+pub struct ExposureSettings {
+    pub aperture_f_stops: f32,
+    pub shutter_speed_s: f32,
+    pub sensitivity_iso: f32,
+}
+
+impl Default for ExposureSettings {
+    fn default() -> Self {
+        Self {
+            aperture_f_stops: 4.0,
+            shutter_speed_s: 1.0 / 250.0,
+            sensitivity_iso: 100.0,
```

Comment:
Also, might be worth providing constants with a couple defaults. For instance, `EXPOSURE_DIRECT_SUNLIGHT`, `EXPOSURE_INDOOR_LIGHTING`, `EXPOSURE_MOONLIGHT`, etc.

Reply:
Agreed!

Reply:
Yeah, unless we had other effects of these parameters they should really be collapsed down into one.

Reply:
Good idea @JMS55 - I was thinking that it would be good to expose (HA!!!) the functionality to provide the parameters for those who it is more familiar. Was first thinking of an enum, but I agree the constructor approach could be nicer.

Reply:
@alice-i-cecile hmm... now that you mention it... the filament docs did talk about how the different parameters affect depth of field (or focal depth - the aperture setting I think, I'm not familiar with cameras beyond physics stuff) and motion blur (shutter speed).

Reply:
That's correct :) At the same time, I don't think we want those baked into our rendering engine by default: motion blur and focal depth are very specific effects that should be used artistically.

We're not simulating a real camera IMO, so we should just expose Ev100 and then provide independent control for things like motion blur and focal depth when we have them.